package.json

```diff
@@ -29,7 +29,7 @@
         "object-extended": "0.0.7",
         "declare.js": "0.0.8",
         "glob": "~3.1.21",
-        "node-etcd": "4.0.x"
+        "node-etcd": "~4.2.1"
```

Comment:
I'm not sure. If we take that stance it's going to be a pain every time we push out a hot fix for something. However this isn't our library so I don't know. Maybe we should discuss. I haven't been doing exact in other OS libraries just C2FO node itself.